SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.spec

```diff
@@ -87,26 +23,39 @@ loading.
 
 %prep
 %autosetup -n shim-%{version} -p1
+# shim Makefile expects vendor SBATs to be in data/sbat.<vendor>.csv
+sed -e "s,@@VERSION_RELEASE@@,%{version}-%{release},g" %{SOURCE1} > ./data/sbat.microsoft.csv
+cat ./data/sbat.microsoft.csv
```

Comment:
Do we need to print the contents here or is this a leftover from debugging? If we'd like to keep this, I'd suggest at least echoing a message earlier, so the log says what is being printed.

Reply:
I'd like to keep this output. The log always writes out the command and stdout so it is pretty clear what is being logged.
```
time="2021-04-01T01:55:34Z" level=debug msg="+ sed -e s,@@VERSION_RELEASE@@,15.4-1.cm1,g /usr/src/mariner/SOURCES/sbat.csv.in"
time="2021-04-01T01:55:34Z" level=debug msg="+ cat ./data/sbat.microsoft.csv"
time="2021-04-01T01:55:34Z" level=debug msg="shim.mariner,1,Microsoft,shim,15.4-1.cm1,https://github.com/microsoft/CBL-Mariner"
time="2021-04-01T01:55:34Z" level=debug msg="+ exit 0"
```